en/starter/generator/command_line_tools_for_linux.md

```diff
@@ -27,7 +27,7 @@ generator and its prerequisites are installed correctly.
 >
 > * [IBM API Connect toolkit CLI](https://www.ibm.com/support/knowledgecenter/en/SSMNED_5.0.0/com.ibm.apic.toolkit.doc/capic_swift_overview.html) via `apic swiftserver`  
 > (uses v1.0.4 of the generator).
-> * [Bluemix CLI Developer Plug-in](https://console.ng.bluemix.net/docs/cloudnative/tutorial_web.html#tutorial/) via `bx dev create`  
+> * [IBM Cloud CLI Developer Plug-in](https://console.ng.bluemix.net/docs/cloudnative/tutorial_web.html#tutorial/) via `bx dev create`  
```

Comment:
According to the this page https://console.bluemix.net/docs/cloudnative/dev_cli.html#developercli we are talking about the IBM Cloud Developer Tools CLI (formerly referred to as the Bluemix CLI Developer Plug-in), so we should match their naming here.

Reply:
Noted, I'll open a new PR with the changes.